assets/webpack.config.js

```diff
@@ -8,10 +8,15 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
-module.exports = (_env, _options) => ({
+const devtoolOption = (mode) => {
+  if (mode === 'production') return 'source-map';
+  return 'eval';
+};
+
+module.exports = (_, {mode}) => ({
   optimization: {
     minimizer: [
-      new TerserPlugin({cache: true, parallel: true, sourceMap: false}),
+      new TerserPlugin({cache: true, parallel: true}),
```

Comment:
https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_assets/webpack.config.js#L15

```js
      new TerserPlugin({cache: true, parallel: true, sourceMap: devMode}),
```

Reply:
The `terser-webpack-plugin` [used](https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_assets/package.json#L24) in the Phoenix generator is a bit old (2.X vs 4.X).

This option is no longer supported.